### ISSUE TITLE
menubar interactions are hacky

### ISSUE THREAD

**Issue by rpearl** [chlorophyll/chlorophyll#111]
Right now the menubar is not nice, and kinda hacked together. We'll need to figure out some interactions between the menubar and various pieces of code, like the toolbox.

**Comment by cwill** [comment]
closing to be replaced by issues describing concrete tasks

**Comment by rpearl** [comment]
I don't agree with closing this? it should be tracked until we have concrete tasks (which will reference this), instead of disappearing until we rediscover that it's a problem

**Comment by cwill** [comment]
Then it needs to be rewritten to describe a problem, intention, or goal because as it stands I cannot glean from the description what actually needs to be addressed other than "it's not good"
